eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,8 +12,8 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [String]$Unreleased=$True,
-  [String]$ReplaceLatestEntryTitle = $False,
+  [String]$Unreleased = "true", #Argument is string becasue of the different ways the script is called in the various repos.
```

Comment:
Curious about this, why can't you do this?

Reply:
This is because of how the script is called [here](https://github.com/Azure/azure-sdk-for-js/blob/cd97a786e858f3166b612ccae450082479f7a600/eng/tools/versioning/set-version.js#L69). PowerShell in converting to Boolean results in true for both "true" and "false"

Reply:
I'd normally use a ```[switch]``` type for this kinds of options in PowerShell, example:

```powershell
function Do-Stuff
{
  param(
    [switch]$FooSwitch
  )

  Write-Host $FooSwitch
}
```

When you invoke this function you can do this:

```
Do-Stuff
Output: False

Do-Stuff -FooSwitch
Output: True

Do-Stuff -FooSwtich:$false
Output: False

Do-Stuff -FooSwitch:$([System.Boolean]::Parse("false"))
Output: False

Do-Stuff -FooSwitch:$([System.Boolean]::Parse("true"))
Output: True
```

In short PowerShell provides the mechanisms to declare these kinds of switches and pass in true-false values.

Reply:
Oh, I see.
Do you know how the switch will be invoked from [JS](https://github.com/Azure/azure-sdk-for-js/blob/cd97a786e858f3166b612ccae450082479f7a600/eng/tools/versioning/VersionUtils.js#L39-L40)

Reply:
Yes I expect the switch to work fine from anywhere (Note that -FooSwitch:0/1 also works). The key with using a switch is to use a term that means the variable defaults to false and so folks will only ever pass it if they want to set it to true.

Reply:
It looks like the array that is passed into the spawn command is just the argument list. But it looks like ```pwsh.exe``` has a nice little affordance where you can invoke it like this:

```
pwsh.exe [script] [args]
```

For example:

```
pwsh.exe Do-Stuff.ps1 -FooSwitch:false
```

(note you don't need to do the [System.Boolean] thing.

Reply:
@weshaggard Unfortunately ` -FooSwitch:0/1` does not work.